docs/reference/document_tree.rst

```diff
@@ -231,14 +231,20 @@ like this:
     .. code-block:: php
 
         <?php echo $view['actions']->render(new ControllerReference(
-                'sonata.admin.doctrine_phpcr.tree_controller:treeAction',
+                'sonata.admin.doctrine_phpcr.tree_controller::tree',
                 array(
                     'root'     => $basePath . '/menu',
                     'selected' => $menuNodeId,
                     '_locale'  => $app->getRequest()->getLocale()
                 ),
         )) ?>
 
+.. note::
+    To use the configuration for Symfony < 3.4 you should use the single colon (:) notation to define controller
+    actions: ``sonata.admin.doctrine_phpcr.tree_controller:treeAction``. Have a look at the `Symfony documentation`_
+    (here at the example for version 3.1).
```

Comment:
i would leave away the link (so the last sentence). and s/you should use/have to use/. with double colon it would simply be broken, so its not just "should"